secrets/README.md

```diff
@@ -1,5 +1,64 @@
 # Secrets
 
+## dynamic-opaque-jinja.j2
+
+You can use this to create secrets in a more dynamic way including different namespaces and multiple key/value pairs.
+The template will use a dict of the following format:
+```
+secrets:
```

Comment:
maybe just say "create a yaml file named vars.yml" and then below change `-e $secrets` to `-e @vars.yml`. then you could even remove the `-i <inventory>` piece. otherwise it works and lgtm